R/pr.R

```diff
@@ -13,7 +13,7 @@
 #' (SSH vs HTTPS) is determined from the existing remote URL(s) of the repo.
 #'
 #' @section For contributors:
-#' To contribute to a package, first use `create_from_github(owner/repo)` to
+#' [To contribute to a package](https://usethis.r-lib.org/articles/articles/pr-functions.html), first use `create_from_github(owner/repo)` to
```

Comment:
Can you add this link in a very obvious way? As more like, "to read more about blah blah blah, see the yada yada vignette".

Reasoning is along these lines:

https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/

Please also follow the line length convention in surrounding lines.